full/build.gradle

```diff
@@ -68,6 +68,11 @@ dependencies {
 
     compile 'org.jsoup:jsoup:1.11.3'
 
+    compile group: 'org.seleniumhq.selenium', name: 'selenium-java', version: '3.141.59'
```

Comment:
Investigate the dimension of selenium in order to provide a mechanism similar to MongoDB drive